From 681e78335437ab932755dd0e9617ee5c9bce0280 Mon Sep 17 00:00:00 2001
From: abelino <abelino.romo@gmail.com>
Date: Thu, 12 Sep 2024 12:38:10 -0700
Subject: [PATCH 2/8] Do not interpret a zero'd out device in a static list as
 valid

---
 src/bacnet/basic/service/h_whois.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/bacnet/basic/service/h_whois.c b/src/bacnet/basic/service/h_whois.c
index 77057065..27656697 100644
--- a/src/bacnet/basic/service/h_whois.c
+++ b/src/bacnet/basic/service/h_whois.c
@@ -126,6 +126,8 @@ static void check_who_is_for_routing(
 
     while (Routed_Device_GetNext(&bcast_net, my_list, &cursor)) {
         dev_instance = Device_Object_Instance_Number();
+        if (dev_instance == 0) continue;
+
         /* If len == 0, no limits and always respond */
         if ((len == 0) ||
             ((dev_instance >= low_limit) && (dev_instance <= high_limit))) {
-- 
2.48.1

