From 0c006a94ea396c6ad0e1f3426ed19eb85df1f199 Mon Sep 17 00:00:00 2001
From: abelino <abelino.romo@gmail.com>
Date: Thu, 12 Sep 2024 12:38:10 -0700
Subject: [PATCH] Do not interpret a zero'd out device in a static list as
 valid

---
 src/bacnet/basic/service/h_whois.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/bacnet/basic/service/h_whois.c b/src/bacnet/basic/service/h_whois.c
index 83d0237b..221ed21b 100644
--- a/src/bacnet/basic/service/h_whois.c
+++ b/src/bacnet/basic/service/h_whois.c
@@ -125,6 +125,8 @@ static void check_who_is_for_routing(uint8_t *service_request,
 
     while (Routed_Device_GetNext(&bcast_net, my_list, &cursor)) {
         dev_instance = Device_Object_Instance_Number();
+        if (dev_instance == 0) continue;
+
         /* If len == 0, no limits and always respond */
         if ((len == 0) ||
             ((dev_instance >= low_limit) && (dev_instance <= high_limit))) {
-- 
2.46.2

