From b0d30d5869fda847059cebd562d7fe8a7836fdfd Mon Sep 17 00:00:00 2001
From: abelino <abelino.romo@gmail.com>
Date: Fri, 18 Oct 2024 12:57:27 -0700
Subject: [PATCH 7/8] Allow BACNET_PROTOCOL_REVISION to be set by user

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 86b929aa..c6e06c2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,7 @@ option(
   "build with uci"
   OFF)
 
-set(BACNET_PROTOCOL_REVISION 19)
+set(BACNET_PROTOCOL_REVISION 19 CACHE STRING "BACnet protocol revision")
 
 if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
-- 
2.48.1

